pkg/certs/cert-inspection/certgraphanalysis/metadata_options.go

```diff
@@ -1,20 +1,81 @@
 package certgraphanalysis
 
 import (
+	"fmt"
+	"strings"
+
 	"github.com/openshift/library-go/pkg/certs/cert-inspection/certgraphapi"
+	corev1 "k8s.io/api/core/v1"
 )
 
-type caBundleRewriteFunc func(caBundle *certgraphapi.CertificateAuthorityBundle)
+const lastRewrittenByAnnotation = "openshift.io/last-rewritten-by"
```

Comment:
followup: annotation to be something like `rewrite.cert-inspection.openshift.io/<rewriter's name>` so we track all ofthem.